crates/sui-analytics-indexer/src/handlers/mod.rs

```diff
@@ -147,11 +147,11 @@ impl ObjectStatusTracker {
 
     fn get_object_status(&self, object_id: &ObjectID) -> Option<ObjectStatus> {
         if self.mutated.contains(object_id) {
-            Some(ObjectStatus::Created)
-        } else if self.deleted.contains(object_id) {
             Some(ObjectStatus::Mutated)
-        } else if self.created.contains(object_id) {
+        } else if self.deleted.contains(object_id) {
             Some(ObjectStatus::Deleted)
+        } else if self.created.contains(object_id) {
+            Some(ObjectStatus::Created)
         } else {
             None
```

Comment:
do we expect any `object_id` to have `None` status here, we can `expect` here otherwise?

Reply:
I don't think so, yeah happy to make it an expect here

Reply:
Actually it could be, depending on how we are calling it i.e. with what `object_id`. For instance, in transaction_object table we invoke it with input objects and it would return None

Reply:
It might be possible if the object is some other status, i.e. unwrapped. But I could be wrong.